specs/binding-http.spec/src/main/resources/META-INF/zilla/http.idl

```diff
@@ -49,4 +49,77 @@ scope http
             HttpHeader[] headers;
         }
     }
+
+    scope event
+    {
+        enum HttpEventType (uint8)
+        {
+            CORS_PREFLIGHT_REQUEST_NOT_ALLOWED (1),
+            CORS_REQUEST_NOT_ALLOWED (2),
+            GUARD_AUTHORIZATION_FAILURE (3),
+            GUARD_AUTHORIZATION_SUCCESS (4),
+            ROUTE_NOT_FOUND (5),
+            ROUTE_FOUND (6)
+        }
+
+        enum Level
+        {
+            INFO,
+            WARNING,
+            ERROR
+        }
+
+        struct HttpEventCommon
+        {
+            int64 timestamp = 0;
+            int64 originId;
+            int64 routedId;
+            int64 initialId;
+            int64 replyId;
+            int64 affinity;
+            int64 authorization = 0;
+            int64 budgetId;
+            int64 traceId;
+        }
+
+        struct HttpEventCorsPreflightRequestNotAllowed extends HttpEventCommon
+        {
+            Level level = WARNING;
+        }
+
+        struct HttpEventCorsRequestNotAllowed extends HttpEventCommon
+        {
+            Level level = WARNING;
+        }
+
+        struct HttpEventGuardAuthorizationFailure extends HttpEventCommon
+        {
+            Level level = WARNING;
+        }
+
+        struct HttpEventGuardAuthorizationSuccess extends HttpEventCommon
+        {
+            Level level = INFO;
+        }
+
+        struct HttpEventRouteNotFound extends HttpEventCommon
+        {
+            Level level = WARNING;
+        }
+
+        struct HttpEventRouteFound extends HttpEventCommon
+        {
+            Level level = INFO;
+        }
```

Comment:
Given that log level is defined by the caller, I think we can omit these subtypes for now.

Reply:
removed